src/index.html

```diff
@@ -5,9 +5,35 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="img">
```

Comment:
in this case you should to refactor code style, see my code suggestions

![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/7735b168-b98f-4125-bec7-9959bf3cd17e)

```suggestion
        class="img"
      >
```